tests/utils/test_commands.py

```diff
@@ -0,0 +1,90 @@
+"""
+Tests for command module
+"""
+from subprocess import CalledProcessError
+
+import pytest
+from cg.utils import Process
+
+
+def test_process():
+    # GIVEN a binary with 'ls'
+    binary = "ls"
+    # WHEN instantiating a Process with the binary
+    process = Process(binary=binary)
+    # THEN assert the instatiation worked as expected
+    assert process.binary == binary
+    assert repr(process) == "Process:base_call:['ls']"
+    assert process.stdout == ""
+
+
+def test_process_run_invalid_command():
+    # GIVEN a binary with 'ls'
+    binary = "false"
+    # WHEN instantiating a Process with the binary
+    process = Process(binary=binary)
+    # THEN assert the instatiation worked as expected
+    with pytest.raises(CalledProcessError):
+        process.run_command()
+
+
+def test_process_run_command_no_params(ls_process):
+    # GIVEN a proces with 'ls' as binary
+    process = ls_process
+    # WHEN running the command without parameters
+    res = process.run_command()
+    # THEN assert it returns valid exit code
+    assert res == 0
```

Comment:
suggest to change magic number to constant

Reply:
Removed